kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -470,23 +465,23 @@ private void handleModemManagerGPSSetup(Device device, Optional<Boolean> enableG
         modemLocation.Setup(MMModemLocationSource.toBitMaskFromMMModemLocationSource(currentLocationSources), false);
     }
 
-    private String getDeviceId(Device device) throws DBusException {
-        NMDeviceType deviceType = getDeviceType(device);
+    protected String getDeviceId(String devicePath) throws DBusException {
```

Comment:
Which device path? OS path? DBus path? If it's dbus path is it for the NM Device path or for the MM Modem path?

```suggestion
    protected String getDeviceId(String nmDeviceDbusPath) throws DBusException {
```